pkg/browsers/console.go

```diff
@@ -376,17 +362,7 @@ func makeDestinationURL(service string, region string) (string, error) {
 
 	serv := ServiceMap[service]
 	if serv == "" {
-		var validServices []string
-		for s := range ServiceMap {
-			validServices = append(validServices, s)
-		}
-		// present the strings in alphabetical order.
-		// Yes, this is a bit of computation - but our arrays are quite small
-		// and this avoids the need to keep the ServiceMap alphabetically sorted when developing Granted.
-		sort.Strings(validServices)
-
-		return "", fmt.Errorf("\nservice %s not found, please enter a valid service shortcut \nValid service shortcuts: [%s]\n", service, strings.Join(validServices, ", "))
-
+		serv = service
```

Comment:
I think it's worth displaying a warning to the user if we don't have the service in our `ServiceMap` mapping. I know this creates more maintenance burden for us but I think it improves the UX for users if they make an accidental typo e.g. `assume -c -s ecc2`.

```suggestion
		color.New(color.FgYellow).Fprintf(color.Error, "[warning] we don't recognize service %s but we'll try and open it anyway (you may receive a 404 page)", service)
		serv = service
```

If you make a typo you get a rather unhelpful 404 page, which might make users think there is an issue with the AWS console rather than a mistyped service.

<img width="648" alt="image" src="https://user-images.githubusercontent.com/17420369/175075161-f12e9709-bc8e-4fea-8f73-7dda87a64607.png">